README.md

```diff
@@ -73,8 +73,7 @@ Note: The above is just an example, most likely you'll want to use an existing t
 
 ## Storage Component
 Zipkin includes a [StorageComponent](zipkin/src/main/java/zipkin2/storage/StorageComponent.java), used to store and query spans and
-dependency links. This is used by the server and those making custom
-servers, collectors, or span reporters. For this reason, storage
+dependency links. This is used by the server and those making collectors, or span reporters. For this reason, storage
```

Comment:
"span reporters" is quite the edge case for the storage component. in hind sight I think we should be more cautious about this in general, perhaps remove it from the top-level readme.

I'm thinking move section about storage component to readme under zipkin-storage, saying the only use case is server-development and while we keep dependencies down etc, the storage and collector apis primarily are in service to server extensions and are not intended for people to nest into traced applications

Reply:
Adrian's suggestion here sounds good to me. Better to get this out of the top-level README so people aren't tempted to try this kind of thing.